app/controllers/user_organization_memberships_controller.rb

```diff
@@ -16,10 +18,25 @@ def organization_memberships
     render json: OrganizationMembershipsRepresenter.new(@memberships).to_json
   end
 
+  def generate_random_password
+    SecureRandom.urlsafe_base64(6)
+  end
+
+  def invite_user(email)
+    password = generate_random_password
+    @user = Authentor.new().create_user email: email, password: password
+    sign_in_url = "https://www.getguesstimate.com"
```

Comment:
can this be a constant in our app?  Seems like it should be in production.rb or the secrets, so it can be configured manually.